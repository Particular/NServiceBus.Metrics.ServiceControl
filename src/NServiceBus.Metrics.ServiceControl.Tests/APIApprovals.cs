using System.Runtime.CompilerServices;
using NServiceBus;
using NServiceBus.Metrics.ServiceControl.Tests;
using NUnit.Framework;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
#if NET452
        [Test]
        [MethodImpl(MethodImplOptions.NoInlining)]
        public void ApproveNServiceBus__NET452()
        {
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(MetricsOptionsExtensions).Assembly);
        TestApprover.Verify(publicApi);
        }
#endif

#if NETCOREAPP2_0
    [Test]
    [MethodImpl(MethodImplOptions.NoInlining)]
    public void ApproveNServiceBus__NETSTANDARD2_0()
    {
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(MetricsOptionsExtensions).Assembly);
        TestApprover.Verify(publicApi);
    }
#endif
}

