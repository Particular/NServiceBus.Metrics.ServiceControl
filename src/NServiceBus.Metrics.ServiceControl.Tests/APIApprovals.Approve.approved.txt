namespace NServiceBus
{
    public class static MetricsOptionsExtensions
    {
        public static void SendMetricDataToServiceControl(this NServiceBus.MetricsOptions options, string serviceControlMetricsAddress, System.TimeSpan interval, string instanceId = null) { }
    }
}